docs/restore/rclone.md

```diff
@@ -0,0 +1,108 @@
+# Restore rclone
+
+This will guide you through restoring object storage.
+
+> [!important]
+> Ensure that all `rclone-sync` CronJobs are suspended so it cannot corrupt the backup!
```

Comment:
Could you add a small snippet on how to do this so it is easy to copy?
Ok to skip this if it feels like too much work.

Reply:
Yeah that's also a good idea.

Reply:
Added the condition so that the main flow is to remove them as it is easier to write steps for. :+1: